runtime/gc_include/ObjectAllocationAPI.hpp

```diff
@@ -33,6 +33,9 @@
 #endif /* OMR_OVERRIDE_COMPRESS_OBJECT_REFERENCES */
 #endif /* OMR_OVERRIDE_COMPRESS_OBJECT_REFERENCES */
 
+/* Generic rounding macro - result is a uintptr_t */
+#define ROUND_U64_TO(granularity, number) (((uintptr_t)(number) + (granularity) - 1) & ~((uintptr_t)(granularity) - 1))
```

Comment:
U64 is not correct in the name - the logic is correct (and used) even on 32bit, too. 
Rename it to either ROUND_TO, or perhaps ROUND_DOWN_TO

BTW, the same logic exists in GC_ObjectModelBase::adjustSizeInBytes() in OMR. Ideally, the macro should be on OMR side and both sides use it (but I guess this ok for now),

Reply:
and I'd flip the positions of the arguments

Reply:
we actually have a macro already: ROUND_UP_TO_POWEROF2 (object heap aligment is 8, what indeed is a power of 2)

Reply:
I thought I remembered OMR having such a macro. Thanks, @amicic for pointing it out.
This should definitely use `ROUND_UP_TO_POWEROF2`.

Reply:
Yes I was going with `ROUND_TO` but ran into some issues with redefinition and namespaces. Cool I didn't know about `ROUND_UP_TO_POWEROF2`, I'll update it to make use of this macro, thanks!